searx/templates/simple/result_templates/videos.html

```diff
@@ -5,7 +5,15 @@
 {% if result.embedded -%}
 <p class="altlink"> &bull; <a class="btn-collapse collapsed media-loader disabled_if_nojs" data-target="#result-video-{{ index }}" data-btn-text-collapsed="{{ _('show video') }}" data-btn-text-not-collapsed="{{ _('hide video') }}">{{ icon('film-outline') }} {{ _('show video') }}</a></p>
 {%- endif %}
-{% if result.content %}<p class="content">{{ result.content|safe }}</p>{% endif %}
+<p class="content">
+  {%- if result.content %}
+    {{ result.content|safe }}
+  {%- else %}
+    <div class='italic_font'>
+      {{ _('This site did not provide any description.')|safe }}
+    </div>
+  {% endif -%}
+</p>
```

Comment:
Until now most (all? at least some?) CSS class are named after their meaning.

If we continue to follow this spirit, I suggest this:
```jinja2
{%- if result.content -%}
  <p class="content">{{ result.content|safe }}</p>
{% - else - %}
  <p class="content empty_content">{{ _('This site did not provide any description.')|safe }}</p>
{%- endif -%}
```

If someone wants an preference to hide the empty content, it can be implemented only with CSS.

Yes, it is also possible to have `.content .italic_font` except when we will have another usage of the `italic_font`.

Reply:
> The class name should be specific to the html element it got put on to and not to the actual CSS attributes... 

As an alternative, instead of `empty_content` it can be `empty_element`, so we can flag other HTML elements with this CSS class ; and it is still about the meaning.

Reply:
> If someone wants an preference to hide the empty content, it can be implemented only with CSS.

done!  .. hope `style.less` is the right place .. otherwise change it up to your meanings.